### DISCUSSION TITLE
Potential Rodauth Rails API + SPA/separate client example

### DISCUSSION THREAD

**Discussion by mdodell** [janko/rodauth-rails#136] [General]
Hi @janko! I love the library, and I hope you don't mind all of the Q/A's I've been asking. I hope that it helps future users and helps increase documentation for the library!

I saw a comment somewhere that you didn't have an example available for a SPA/stand-alone client app calling a Rails Rodauth API for authentication, and that it was a bit too much to get into. 

I was able to create an MVP of this, which I'm hosting on Github:
- [API](https://github.com/mdodell/rodauth-linkedin-api)
- [Very barebones NextJS Client](https://github.com/mdodell/rodauth-linkedin-client)

The client is just handling the Omniauth request and logging out, but I'd be happy to create a better demo with all of the basic Rodauth Rails features in a demo app if that would be of any interest.

**Comment by janko** [comment]
Looks nice, thanks for sharing it 👍🏻. Yes, I think it would be interesting for others to see an example of this, if you do add other authentication features, please send a PR to add a link to the rodauth-rails README 😉
